src/assets/Pokemon.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+
+function Pokemon( { url }) {
+
+    const [pm, setPm] = useState(null);
```

Comment:
Het is beter om een beschrijvende naam te gebruiken, als ik dit snel lees en `pm` zie staan, heb ik geen idee wat er in deze data zit